app/models/work.rb

```diff
@@ -0,0 +1,34 @@
+class Work < ApplicationRecord
+  validates :category, presence: true
+  validates :title, presence: true, uniqueness: true
+
+  def self.albums
+    where(category: "album")
+  end
+  def self.books
+    where(category: "book")
+  end
+  def self.movies
+    where(category: "movie")
+  end
+
+  def self.spotlight
+    # all.first
+    spotlight_criteria = self.maximum("publication_year")
+    spotlight = self.find_by(publication_year: spotlight_criteria)
+  end
+
+  def self.top(category)
+    full_collection = Work.where(category: category).order("lower(publication_year) DESC")
```

Comment:
Good that you are using the DB to sort things, odd that you're sorting by publication year.